governance-exercises/Interpretation Exercise Response.md

```diff
@@ -0,0 +1,46 @@
+
+**12.1: Bootstrapping Governance Votes**
+
+Governance Facilitators are empowered to use broad judgment when exercising their ability to make direct Atlas interpretations and edits to the Atlas and the Scope Artifacts, when this relates to unintended consequences or mistakes in the affected documents, *as well as when this relates to time-sensitive issues with the Launch Project.*
+
+12.1.1: In case of unintended consequences or mistakes in the Atlas and Scope Artifacts causing Maker Governance to function incorrectly, the Governance Facilitators can at any time run a Governance Poll to enable MKR holders to make a decision that will alleviate and resolve unintended consequences or mistakes.
+
+12.1.2: The Governance Facilitators can at any time propose to edit any content of a Scope Artifact through a Governance Poll.
+
+https://mips.makerdao.com/mips/details/MIP113#12-1-bootstrapping-governance-votes
+
+
+**Situation A**
+
+**Aligned Delegates are not being compensated due to a clear typo elsewhere in the Governance Scope.**
+
+Fixing the typo is within the power of 12.1. This is clearly a "mistake". if participants who are supposed to be paid cannot be paid. I would create polls(1 or 2, depending on the specifics) that fix the typo and backpay the delegates. I would notify Governance through the forums and Discord.
+
+**Situation B**
+
+**A large token holder wants to add logic to the Stability Scope scope more quickly than is possible using the monthly governance cycle. The author claims this change is required to prevent the DAO losing significant value.**
+
+While this is not clearly included in 12.1, there is precedent that the vote should proceed in this case. I would consider the proposal carefully, along with the governance ramifications of posting the poll or rejecting the token holder's request. As noted above, I would also advise changing the wording to clarify this.
+
+If I did decide to proceed, I would proceed in the same manner as  situation A.
+
+Source: https://vote.makerdao.com/polling/QmcTRPLx#poll-detail and the related forum and Discord discussion
+
+**Situation C**
+
+**An anonymous account wants to add logic to the Stability Scope scope more quickly than is possible using the monthly governance cycle. The author claims this change is required to prevent the DAO losing significant value.**
+
+This situation is more one of philosophy and politics than of Scopes Interpretation. The scopes give no reason to treat this differently than section B, but allowing anybody to propose weekly changes risks flooding governance with polls.
+
+I would still give the proposal consideration, but I would want a stronger consensus from other stakeholders before proceeding.
```

Comment:
The point of exercises is to make you consider it now! I know its a very abstract scenario, but given that, think about what you'd do with various specifics, what are the commonalities in your responses across your imagined specifics? What general themes are important? How do you deliver consistent interpretation across a variety of possibilities?